src/equations/compressible_euler_2d.jl

```diff
@@ -817,6 +817,25 @@ See also
 end
 
 
+@inline function max_abs_speed_naive(u_ll, u_rr, orientation, equations::CompressibleEulerEquations2D)
```

Comment:
Maybe a comment for easier understanding while reading the code as a new user (I am aware of the existence of the docstring)
```suggestion
# Calculate maximum wave speed for local Lax-Friedrichs-type dissipation as the maximum velocity magnitude plus the maximum speed of sound
@inline function max_abs_speed_naive(u_ll, u_rr, orientation, equations::CompressibleEulerEquations2D)
```
We should not forget to add this to MHD and other relevant equations.

Reply:
Sure. just wanted to get your general feedback on this approach before implementing the remaining stuff for other dimensions/equations etc. I added the comment in 287f4d63.